docs/understanding-results.md

```diff
@@ -0,0 +1,248 @@
+# Understanding the Results
+
+The results object contains all the audit information Lighthouse determined about the page. In fact, everything you see in the HTML report, even the screenshots, is a rendering of information contained in the results object. You might need to work directly with the results object if you use [Lighthouse programmatically](https://github.com/GoogleChrome/lighthouse/blob/master/docs/readme.md#using-programmatically), consume the JSON output of the [CLI](https://github.com/GoogleChrome/lighthouse#using-the-node-cli), explore [Lighthouse results in HTTPArchive](https://github.com/GoogleChrome/lighthouse#lighthouse-integrations), or work on the report generation code that reads the Lighthouse JSON and outputs HTML.
+
+## Top-level Object
```

Comment:
can we call this

> Lighthouse Result (LHR) object 

a unique name makes it a bit easier to discuss

Reply:
good call, I knew we had a name for it I was forgetting!